packages/iota/src/models/payloads/IMilestonePayload.ts

```diff
@@ -24,37 +24,37 @@ export interface IMilestonePayload extends ITypeBase<7> {
     timestamp: number;
 
     /**
-     * The parents where this milestone attaches to.
+     * The timestamp of the milestone.
      */
-    parentMessageIds: string[];
+    lastMilestoneId: string;
 
     /**
-     * The merkle proof inclusions.
+     * The parents where this milestone attaches to.
      */
-    inclusionMerkleProof: string;
+    parentMessageIds: string[];
 
     /**
-     * The next PoW score.
+     * The merkle proof confirmation.
```

Comment:
Here we could give a better description taken from TIP-29, i.e. 
`The Merkle tree hash of the Message IDs of all messages confirmed by this milestone.` or 
`The Merkle tree hash of all messages confirmed by this milestone.`